getting_started/getting_started.py

```diff
@@ -10,32 +10,34 @@
     os.chdir("getting_started")
 
 # %% [markdown]
-# ## Programmatically Building & Managing Training Data with Snorkel
+# ## Programmatically Building and Managing Training Data with Snorkel
 #
-# Snorkel is a system for _programmatically_ building and managing training datasets **without needing to hand-label _any_ training data**.
-# In Snorkel, users can develop training datasets in hours or days rather than hand-labeling them over weeks or months.
+# Snorkel is a system for _programmatically_ building and managing training datasets **without manual labeling**.
+# In Snorkel, users can develop large training datasets in hours or days rather than hand-labeling them over weeks or months.
 #
 # Snorkel currently exposes three key programmatic operations:
-# - **Labeling data**, for example using heuristic rules or distant supervision techniques
-# - **Transforming data**, for example rotating or stretching images to perform data augmentation
-# - **Slicing data** into different critical subsets.
+# - **Labeling data**, e.g., using heuristic rules or distant supervision techniques
+# - **Transforming data**, e.g., rotating or stretching images to perform data augmentation
+# - **Slicing data** into different critical subsets for monitoring or target improvement
+#
 # Snorkel then automatically models, cleans, and integrates the resulting training data using novel, theoretically-grounded techniques.
 
 # %% [markdown]
 # <img src="img/Overview.png" onerror="this.onerror=null; this.src='/doks-theme/assets/images/layout/Overview.png';" align="center" style="display: block; margin-left: auto; margin-right: auto;">
 
 # %% [markdown]
-# In this quick walkthrough, we'll preview the high level workflow and interfaces of Snorkel using a canonical machine learning problem: classifying spam.
-# We'll use a public [YouTube comments dataset](https://www.kaggle.com/goneee/youtube-spam-classifiedcomments) from Kaggle, and see how **Snorkel can enable you to train a machine learning model without _any_ hand-labeled training data!**
-# For the more detailed version, see the [Introductory Tutorial](#).
-#
+# In this quick walkthrough, we'll preview the high-level workflow and interfaces of Snorkel using a canonical machine learning problem: classifying spam.
+# We'll use a public [YouTube comments dataset](http://www.dt.fee.unicamp.br/~tiago//youtubespamcollection/), and see how **Snorkel can enable training a machine learning model without _any_ hand-labeled training data!**
+# For more detailed versions of the sections in this walkthrough, see the corresponding tutorials: ([Spam LFs](http://snorkel.org/use-cases/01-spam-tutorial), [Spam TFs](http://snorkel.org/use-cases/02-spam-tutorial), [Spam SFs](http://snorkel.org/use-cases/03-spam-tutorial)).
```

Comment:
Should these links be "Intro" instead of "Spam"?

Reply:
We should standardize @bhancock8 @paroma

Reply:
I like "Intro"

Reply:
Ok, I vote we leave the directory as `spam` so it matches all the others, but we rename the files to `01_lfs_intro_tutorial`, `02_tfs_intro_tutorial`, `03_sfs_intro_tutorial` and we adjust the first header in each file to match (e.g., `Snorkel Introduction to LFs Tutorial: Spam Detection`), and we can refer to them in the text then as the "LFs Introduction tutorial", etc.